### PR TITLE
Update to SE-onboarding-car

### PR DESCRIPTION
**Pull request by simonauner**
This PR adds `SE-onboarding-car` flow:

image

It's a merge of `Web Onboarding SE - Quote Cart` and `SE-cross-sell-car`

https://hedvig.atlassian.net/browse/GRW-1000

### CODE REVIEW

**Comment by simonauner** [comment]
If see now that we're not asking for whether the user currently has a car insurance or not, I will add that!

File: angel-data/SE-onboarding-car.json [hunk 1]
```diff
@@ -0,0 +1,151 @@
+{
+  "computedStoreValues": [
+    {
+      "key": "insuranceType",
+      "value": "'SWEDISH_CAR'"
```
Comment (robinandeer) [inline angel-data/SE-onboarding-car.json:5]:
```suggestion
      "value": "SWEDISH_CAR"
```

**Review by robinandeer — APPROVED** [review]

File: angel-data/SE-onboarding-car.json [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      "tags": [],
+      "text": "<Redirect to=\"[[CreateQuoteCartError]]\"\n          when=\"errorMessage != null\"\n>\n</Redirect>\n\n<Message>\n  Your offer is ready!\n</Message>\n\n<QuoteCartOfferRedirect id=\"quoteCartId\"\n                        when=\"true\"\n                        insurancetypes=\"insuranceType\"\n>\n</QuoteCartOfferRedirect>",
+      "top": 1276,
+      "width": 100
+    }
+  ],
+  "script": "",
+  "snapToGrid": false,
+  "startPassage": "28e0a0fe-b56d-4efe-863c-670f6a68150e",
+  "storyFormat": "Hedvig - Development",
```
Comment (simonauner) [inline angel-data/SE-onboarding-car.json:146]: Should this be just `"Hedvig"`?

Comment (robinandeer) [reply angel-data/SE-onboarding-car.json:146]: It doesn't really matter :)

**Review by robinandeer — DISMISSED** [review]

**Review by robinandeer — APPROVED** [review]

**Pull request merged**